mne_bids/tests/test_io.py

```diff
@@ -0,0 +1,29 @@
+"""Testing utilities for file io."""
+# Authors: Stefan Appelhoff <stefan.appelhoff@mailbox.org>
+#
+# License: BSD (3-clause)
+import pytest
+
+from mne_bids.io import _parse_ext, _read_raw
+
+
+def test__parse_ext():
```

Comment:
accidental double underscore

Reply:
That was intentional (but maybe wrong): I was following the convention of defining a test function "fun" as `def test_fun`. In this case, fun was `_parse_ext`, thus `test_` + `_parse_ext` = double underscore. I take from your comment that this is not the convention?

Reply:
oh well, I never followed this convention ...but as you like